### PR TITLE
Deploy ag fix folders2

### PR DESCRIPTION
**Pull request by alexgarel**
* fix core dump of annoy
* better dev setup
* data folder is now a parameter

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -24,7 +24,10 @@
 
 
 def init_sentry(integrations: Sequence[Integration] = ()):
-    if _sentry_dsn:
```
Comment (mithridatea) [inline settings.py:27]: a simpler way to disable sentry for dev is to not provide `_sentry_dns` (or having a null value): https://docs.sentry.io/platforms/python/configuration/options/#dsn

File: docker/prod.yml [hunk 2]
```diff
@@ -1,5 +0,0 @@
-version: "3.9"
```
Comment (mithridatea) [inline docker/prod.yml:1]: This file is going to be added back once we're sure it works well on dev, right?

Comment (alexgarel) [reply docker/prod.yml:1]: no, we don't need it anymore

File: settings.py (hunk 1 above)
Comment (alexgarel) [reply settings.py:27]: yep, but our code was raising, that why I put a disabled. At least this is explicit when you don't want sentry, and not that you forgot it.

**Review by mithridatea — APPROVED** [review]

**Pull request merged**